zulip_bots/zulip_bots/test_lib.py

```diff
@@ -103,12 +104,20 @@ def call_request(self, message, expected_method, response, state_handler):
         # type: (Dict[str, Any], str, Dict[str, Any], Optional[StateHandler]) -> None
         if state_handler is None:
             state_handler = StateHandler()
+
+        instance = self.MockClass.return_value
+
+        # Set mock identity of the bot
+        name = "test bot"
+        pad = '**' if ' ' in name else ''
```

Comment:
We might just always want to wrap a bot's name in `**`, since it's more consistent and also gives a name a nice @-mention  (or bold) formatting.

Reply:
I'm increasingly feeling that the 'padded' form is really just for mentioning, so I'm inclined to adjust this to providing a `mentioned_name` or similar which even includes the `@` prefix.

Re the unpadded/mentioned name, or even email, we could leave it/them out, but if so then keeping an interface like BotIdentity could be useful?

Having an unquoted name could be useful, in terms of "introducing" themselves without enforcing styling, eg.
```
Hello, you can call me *bot_name*; I'm a bot_type,
so try some of the following to interact with me:
    quoted_name help - show my help text
...
```

How does that sound?